### DISCUSSION TITLE
Boot without SD card

### DISCUSSION THREAD

**Discussion by M4rc311o** [Joshua-Riek/ubuntu-rockchip#255] [Q&A] [answered]
Just a one quick question, everything is working fine when I boot from SD card. I flashed image to my SATA SSD as in Wiki. Orange Pi is booting from SSD now, but it's unable to boot when the SD card is not inserted. Is there something what can I do about that so I can boot from SSD without need of SD card? Thanks in advance.

**Comment by M4rc311o** [comment, marked as answer]
nvm; You just have to mount sda1 and edit _overlays_ in _ubuntuEnv.txt_ to "sata".

**Reply by guynich** [reply]
Thanks for this!  Here's an alternative install method that worked for me using OrangePi 5 (8GB).  

Similarly I followed the Wiki instructions: after power down then removing the microSD and rebooting the board, I saw the orange Ubuntu logo and heard a sound but the nvme SSD boot got stuck.  Seems the same as OP?

I repeated the whole install like this:

1) with microSD (JR v1.18 Desktop) and SSD attached, I used Chromium web browser to download the release file, and the SHA256 file to default Downloads folder.  Then I checked the integrity like this:
```
cd ~/Downloads
sha256sum -c *.sha256
```

2) copied the release from microSD to nvme using `sudo` like this:
```
sudo xz -dc ~/Downloads/ubuntu-22.04.2-preinstalled-desktop-arm64-orangepi5.img.xz | sudo dd of=/dev/nvme0n1 bs=4k status=progress

sudo sync
```

3) powered down the board using Ubuntu gui, removed the microSD, and booted from nvme

This time Ubuntu booted with the desktop gui setup.  FYI: I checked the `UbuntuEnv.txt` file but found `overlays=`  is blank.  E.g.:
```
sudo nano /boot/firmware/ubuntuEnv.txt
```
